avro-ts-cli: Cannot find module 'ansi-regex'

It is not possible to run the CLI without manually installing 'ansi-regex'.
This applies both when running through npx or when installing it. 
```
npx @ovotech/avro-ts-cli ./*.avsc --output-dir .\typescript\
npx: installed 12 in 8.231s
Cannot find module 'ansi-regex'
```

--- a/packages/avro-ts-cli/package.json
+++ b/packages/avro-ts-cli/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@ovotech/avro-ts-cli",
   "description": "A cli to convert avro schemas into typescript interfaces",
-  "version": "2.1.2",
+  "version": "2.1.3",
   "main": "dist/index.js",
   "source": "src/index.ts",
   "types": "dist/index.d.ts",
@@ -25,12 +25,12 @@
   "dependencies": {
     "@ovotech/avro-ts": "^4.1.2",
     "chalk": "^4.0.0",
-    "commander": "^5.0.0"
+    "commander": "^5.0.0",
+    "ansi-regex": "^5.0.0"
   },
   "devDependencies": {
     "@types/jest": "^25.2.3",
     "@types/node": "^14.0.5",
-    "ansi-regex": "^5.0.0",
     "jest": "^26.0.1",
     "prettier": "^2.0.5",
     "ts-jest": "^26.0.0",

